### DISCUSSION TITLE
Lab4-1,Lab4-2, synthesis

### DISCUSSION THREAD

**Discussion by JackeyUi** [bol-edu/HLS-SOC-Discussions#121] [Q&A] [answered]
想確認一下，請問在 Lab4-1 和 Lab4-2 做 synthesis report，都只需要做 User-project的部分(/rtl/user/ )就好了嗎？
還是要把整個 caravel-soc 包一起做呢？

**Comment by LingChiYang** [comment, marked as answer]
同學好

只需要做user_project + bram

**Reply by JackeyUi** [reply]
好的，另外想請問，在 Lab4-1 中有提到 wb_ack 要晚10個clocks回應，但那張示意圖是在 wb_stb 跟 wb_cyc rise起來後的第12個 clock wb_ack 才反應，請問要照哪邊去做？

**Reply by LingChiYang** [reply]
由於有bram，所以若Delay=0 也還需要2 cycle respond。 照著波型是Delay + 2
